tune/protox/agent/off_policy_algorithm.py

```diff
@@ -182,7 +184,9 @@ def collect_rollouts(
             # Rescale and perform action
             new_obs, rewards, terms, truncs, infos = env.step(actions)
             dones = terms or truncs
-            if self.logger:
+            # We only stash the results if we're not doing HPO, or else the results from concurrent HPO would get
+            #   stashed in the same directory and potentially cause a race condition.
+            if self.logger and not tuning_mode == TuningMode.HPO:
```

Comment:
Can we also use the `ray_trial_id` to stash the results during HPO?

Reply:
Thanks for catching that!